partcad/src/partcad/assembly_config.py

```
@@ -21,3 +22,8 @@ def normalize(name, config):
             config = {"type": "alias", "source": config}
 
         return ShapeConfiguration.normalize(name, config)
+
+    @staticmethod
+    def get_manufacturing_data(assembly):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `get_manufacturing_data`

(ANN205)